src/components/application_manager/src/commands/hmi/response_from_hmi.cc

```diff
@@ -72,12 +73,8 @@ void ResponseFromHMI::SendResponseToMobile(
 void ResponseFromHMI::CreateHMIRequest(
     const hmi_apis::FunctionID::eType& function_id,
     const smart_objects::SmartObject& msg_params) const {
-  smart_objects::SmartObjectSPtr result = new smart_objects::SmartObject;
-
-  if (!result) {
-    SDL_ERROR("Memory allocation failed.");
-    return;
-  }
+  smart_objects::SmartObjectSPtr result =
+      utils::MakeShared<smart_objects::SmartObject>();
```

Comment:
@vlantonov , in make shared : in case NULL pointer after new we just return empty SPtr. In SPtr we check NULL ptr with DCHECK, and app always must be crashed. Because if we will use return from check to NULL, we will create incorrect behavior of SDL which not correspond to requirements. 
This behavior of app tell to us about problems, and we must investigate in case incorrect allocation of memory.